src/lib/Search/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -236,10 +237,19 @@ private function getContentInfoList(
                 'c',
                 LocationGateway::CONTENT_TREE_TABLE,
                 'main_tree',
-                $query->expr()->andX(
+                $expr->andX(
```

Comment:
`andX` is deprecated.
```suggestion
                $expr->and(
```